src/network/protocol/messages/getdata.zig

```diff
@@ -0,0 +1,165 @@
+const std = @import("std");
+const CompactSizeUint = @import("bitcoin-primitives").types.CompatSizeUint;
+
+const Sha256 = std.crypto.hash.sha2.Sha256;
+
+const protocol = @import("../lib.zig");
+
+pub const GetdataMessage = struct {
+    inventory: []const InventoryItem,
+
+    pub const InventoryItem = struct {
+        type: u32,
+        hash: [32]u8,
+    };
```

Comment:
This will be shared between `inv` and `getdata` message so can you take it out of there and put it in messages/lib.zig for now

Reply:
ok